requirements.txt

```diff
@@ -5,4 +5,6 @@ sphinx-proof
 matplotlib
 pyccel 
 ipykernel
-git+https://github.com/pyccel/psydac.git
+notebook
+nb-clean
+psydac[extra] @ git+https://github.com/pyccel/psydac.git@devel#egg=psydac
```

Comment:
What does this `[extra]` mean?

BTW the end of file is missing a newline character.

Reply:
`[extra]` is a [dependency specifier](https://packaging.python.org/en/latest/specifications/dependency-specifiers/#extras) which basically installs optional features of a package. I have removed it since the `jupyterbook build` seems to succeed even without it.